lib/client.c

```diff
@@ -152,136 +147,6 @@ static void log_lock_func(void *udata, int lock) {
     }
 }
 
-/**
```

Comment:
oh, you no longer need this function?

Reply:
Yes, this function was broken up into two other functions (`snowflake_timestamp_from_epoch_seconds` and `snowflake_timestamp_to_string`) when I did the timestamp changes. I kept that function around incase I needed to reference it later, but I can just look back at an old commit if I want to reference this function so I thought I would just reduce the code in `client.c` since it is becoming quite the large file.